docs/CHANGELOG.asciidoc

```diff
@@ -77,6 +78,352 @@ Review important information about the {kib} 8.x releases.
+* Log rate analysis: ensures ability to sort on Log rate change ({kibana-pull}193501[#193501]).
+* Single Metric Viewer: Enables cross-filtering for 'by', 'over', and 'partition' field values ({kibana-pull}193255[#193255]).
+* Adds link to anomaly detection configurations from Integration > Assets tab ({kibana-pull}193105[#193105]).
+* Anomaly Explorer: Displays markers for scheduled events in distribution-type anomaly charts ({kibana-pull}192377[#192377]).
+* Serverless Security: Adds ES|QL visualizer menu item to the nav ({kibana-pull}192314[#192314]).
+* Updates icons for Machine Learning embeddable dashboard panel types ({kibana-pull}191718[#191718]).
+* AIOps: Uses no minimum time range by default for pattern analysis ({kibana-pull}191192[#191192]).
+* Links to ML assets from Integration > Assets tab ({kibana-pull}189767[#189767]).
+* Utilizes the `DataViewLazy` in ML plugin ({kibana-pull}189188[#189188]).
+* AIOps: Chunks groups of field candidates into single queries for top items and histograms ({kibana-pull}189155[#189155]).
+* AIOps: Updates fields filter popover to be able to filter fields from analysis (not just grouping) ({kibana-pull}188913[#188913]).
+* Single Metric Viewer embeddable: adds forecasting ({kibana-pull}188791[#188791]).
+* Adds new custom rule action to force time shift ({kibana-pull}188710[#188710]).
+* AIOps: Chunks groups of field candidates into single queries ({kibana-pull}188137[#188137]).
+* AIOps: Adds log rate analysis to alert details page contextual insight ({kibana-pull}187690[#187690]).
+* Adds ability to toggle visibility for empty fields when choosing an aggregation or field in Anomaly detection, data frame analytics ({kibana-pull}186670[#186670]).
+* Anomaly Detection: Adds popover links menu to anomaly explorer charts ({kibana-pull}186587[#186587]).
+Management::
+* Adds an option to show or hide empty fields in dropdown lists in Transform ({kibana-pull}195485[#195485]).
+* Adds a confirmation dialog when deleting a transform from a warning banner ({kibana-pull}192080[#192080]).
+* Improves the autocomplete to suggest fields for the `dense_vector` type in Console ({kibana-pull}190769[#190769]).
+* Adds the ability to view an ILM policy details in read-only mode ({kibana-pull}186955[#186955]).
+
+[float]
+[[fixes-v8.16.0]]
+=== Bug fixes
+Alerting::
+* Skip scheduling actions for the alerts without scheduledActions ({kibana-pull}195948[#195948]).
+* Fixes Stack Alerts feature API access control ({kibana-pull}193948[#193948]).
+* Remove unintended internal Find routes API with public access ({kibana-pull}193757[#193757]).
+* Convert timestamp before passing to validation ({kibana-pull}192379[#192379]).
+* Grouped over field is not populated correctly when editing a rule ({kibana-pull}192297[#192297]).
+* Mark slack rate-limiting errors as user errors ({kibana-pull}192200[#192200]).
+* Use absolute time ranges when adding visualizations to a case ({kibana-pull}189168[#189168]).
```

Comment:
As mentioned above let's have a dedicated section for Cases.

Reply:
I've created https://github.com/elastic/kibana-release-notes/pull/33 to affect the release note generator

Reply:
I've manually added that section now via b20d638e7ddf062aece7999254002bae5b12d984